app/_locales/en/messages.json

```diff
@@ -1106,6 +1139,9 @@
   "grantedToWithColon": {
     "message": "Granted to:"
   },
+  "gweiInParanthesis": {
+    "message": "(GWEI)"
```

Comment:
Does GWEI need to be translated? I feel like it's a universal concept.

Reply:
Fixed